server/render.js

```diff
@@ -1,238 +1,4 @@
-        <link rel="icon" href="/icon-256x256.png" sizes="256x256"/>
-        <link rel="apple-touch-icon" href="/icon-256x256.png" sizes="256x256"/>
-        <link rel="icon" href="/icon-192x192.png" sizes="192x192"/>
-        <link rel="apple-touch-icon" href="/icon-192x192.png" sizes="192x192"/>
-        <link rel="icon" href="/icon-96x96.png" sizes="96x96"/>
-        <link rel="apple-touch-icon" href="/icon-96x96.png" sizes="96x96"/>
-        <link rel="icon" href="/icon-48x48.png" sizes="48x48"/>
-        <link rel="apple-touch-icon" href="/icon-48x48.png" sizes="48x48"/>
-        <link rel="manifest" href="${manifest}">
-
-        <meta name="apple-mobile-web-app-capable" content="yes"/>
-        <meta name="mobile-web-app-capable" content="yes"/>
-        <meta name="apple-mobile-web-app-title" content="Abakus"/>
-
-        <link href="https://cdn.jsdelivr.net/npm/font-awesome@4.7.0/css/font-awesome.min.css" rel="stylesheet">
-        <link href="https://unpkg.com/ionicons@3.0.0/dist/css/ionicons.min.css" rel="stylesheet">
-        <link href="https://fonts.googleapis.com/css?family=Cardo|Raleway|Roboto" rel="stylesheet">
-
-        ${helmet.meta.toString()}
-
-        ${styles}
-      </head>
-      <body>
-        <div id="root">${body}</div>
-        <script>
-           window.__CONFIG__ = ${serialize(config, { isJSON: true })};
-           window.__PRELOADED_STATE__ = ${serialize(state, { isJSON: true })};
-        </script>
-        ${
-          config.environment === 'production'
-            ? `<script>
-
-  !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","debug","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t,e){var n=document.createElement("script");n.type="text/javascript";n.async=!0;n.src="https://cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var a=document.getElementsByTagName("script")[0];a.parentNode.insertBefore(n,a);analytics._loadOptions=e};analytics.SNIPPET_VERSION="4.1.0";
-          analytics.load("${config.segmentWriteKey}");
-          }}();
-      </script>`
-            : ''
-        }
-        <script async src="https://js.stripe.com/v2/"></script>
-        <script async src="https://js.stripe.com/v3/"></script>
-        ${dllPlugin}
-        ${scripts}
-      </body>
-    </html>
-   `;
-}
-
-export default render;
+const noSSR = (req, res) => res.send(pageRenderer());
+export default (__DEV__ ? noSSR : require('./ssr.js').default);
```

Comment:
Since this doesn't import './ssr.js' in development, it doesn't have to compile all the millions of lines with js (since we don't use the SSR in dev, heh)